index.js

```diff
@@ -176,4 +191,33 @@ class Client {
   }
 }
 
+// tracing takes a config and returns a function with the signature
```

Comment:
The comments are great here.
But it makes more sense to use JSDocs here, as it is common practice for oss js projects: http://usejsdoc.org/